lib/puppet/provider/package/pkgin.rb

```diff
@@ -12,8 +12,8 @@
   def self.parse_pkgin_line(package)
 
     # e.g.
-    #   vim-7.2.446 =        Vim editor (vi clone) without GUI
-    match, name, version, status = *package.match(/(\S+)-(\S+)(?: (=|>|<))?\s+.+$/)
+    #   vim-7.2.446;Vim editor (vi clone) without GUI
+    match, name, version, status = *package.match(/(\S+)-(\S+?);(?:(=|>|<))?.+$/)
```

Comment:
Is this a backwards compatible change?  That is to say, was this always broken or did pkgin's format change?